drupal/acquia/commit_push_branch_dev.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# This script checks to see if a user's working tree is not clean
+# Next is adds, commits and pushes all working changes to the Dev Enviroment
+#
+#
+# $ sh ~/Sites/_hal/drupal/acquia/commit_push_branch_dev.sh
+#
+# Notes
+# - Be sure you run from ~/Sites/devdesktop/DESIRED_CODEBASE/docroot/sites
+#
+#
+# Dependencies:
+# - Drush: https://www.drush.org/
+# - Acquia CLI: https://github.com/typhonius/acquia_cli
+#
+# Paramaters:
+# - Branch_Name  | 'Branch Name'
+
+
+
+echo "This script will add, commit and push all selected branches to acquia."
+echo "Please Enter the  Branch name"
+read Branch_Name
```

Comment:
please change Branch_Name to BRANCH_NAME (all caps) throughout script. Not technically needed, but will help match style of other scripts.

Reply:
I Capitalized BRANCH_NAME on the latest commit